build.gradle

```diff
@@ -789,14 +789,19 @@ def copyTestsCommonResources(def bdir) {
   }
 }
 
-def runScript(def execName, def workDir, def param) {
+def runScript(def execName, def workDir, def param, env = null) {
   def stdout = new ByteArrayOutputStream()
+  if (null == env) {
+    env = new HashMap<String,Object>()
+  }
+  env.put('PYTHONPATH', "${snappyProductDir}/python/lib/py4j-0.10.1-src.zip:${snappyProductDir}/python")
```

Comment:
Is this being done in any of our scripts? We need to ensure that python examples can be easily run using our scripts and not just using gradle.